src/test/java/com/ipiecoles/java/java350/model/EmployeTest.java

```diff
@@ -145,6 +147,118 @@ public void testGetPrimeAnnuelle(String matricule, Double tempsPartiel, Integer
 			
 		}
 		
+		@Test
+		public void testAugmenterSalairePourcentage() throws EmployeException {
+				
+				//Given
+				Double pourcentage = 20d;
+				Employe employe = new Employe();  
+				employe.setSalaire(1000d);
+				
+				//When
+				employe.augmenterSalaire(pourcentage);
+				
+				//Then
+				Double salaire = employe.getSalaire();
+				Assertions.assertThat(employe.getSalaire()).isEqualTo(1200d);
+			
+		}
+		
+		@Test
+		public void testAugmenterSalairePourcentage0() throws EmployeException {
+				
+				//Given
+				Double pourcentage = 0d;
+				Employe employe = new Employe(); 
+				employe.setSalaire(1000d);
+				
+				//When
+				employe.augmenterSalaire(pourcentage);
+				
+				//Then
+				Double salaire = employe.getSalaire();
+				Assertions.assertThat(employe.getSalaire()).isEqualTo(salaire);

```

Comment:
Ce test est inutile, car en gros tu fais : `Assertions.assertThat(employe.getSalaire()).isEqualTo(employe.getSalaire());`
```suggestion
				Assertions.assertThat(employe.getSalaire()).isEqualTo(1000d);
```